meta-avocado-raspberrypi/recipes-bsp/bootfiles/rpi-config_git.bbappend

```diff
@@ -4,26 +4,30 @@ do_deploy:append:seeed() {
     grep -q "^dtoverlay=dwc2,dr_mode=host$" $CONFIG || echo "dtoverlay=dwc2,dr_mode=host" >> $CONFIG
     grep -q "^enable_uart=1$" $CONFIG || echo "enable_uart=1" >> $CONFIG
     grep -q "^dtparam=spi=on$" $CONFIG || echo "dtparam=spi=on" >> $CONFIG
+    grep -q "^initramfs avocado-initramfs.cpio.zst followkernel$" $CONFIG || echo "initramfs avocado-initramfs.cpio.zst followkernel" >> $CONFIG
 
-    if ${@bb.utils.contains('MACHINEOVERRIDES', 'seeed-reterminal-dm', 'true', 'false', d)} \
-        || ${@bb.utils.contains('MACHINEOVERRIDES', 'seeed-reterminal-dm-mender', 'true', 'false', d)}; then
+    if ${@'true' if 'seeed-reterminal-dm' in d.getVar('MACHINEOVERRIDES').split(':') else 'false'} \
+        || ${@'true' if 'seeed-reterminal-dm-mender' in d.getVar('MACHINEOVERRIDES').split(':') else 'false'}; then
         grep -q "^dtoverlay=reTerminal-DM$" $CONFIG || echo "dtoverlay=reTerminal-DM" >> $CONFIG
         grep -q "^dtparam=i2c_vc=on$" $CONFIG || echo "dtparam=i2c_vc=on" >> $CONFIG
         grep -q "^dtoverlay=i2c3,pins_4_5$" $CONFIG || echo "dtoverlay=i2c3,pins_4_5" >> $CONFIG
-    elif ${@bb.utils.contains('MACHINEOVERRIDES', 'seeed-reterminal', 'true', 'false', d)} \
-        || ${@bb.utils.contains('MACHINEOVERRIDES', 'seeed-reterminal-mender', 'true', 'false', d)} \
-        || ${@bb.utils.contains('MACHINEOVERRIDES', 'dual-gbe-cm4', 'true', 'false', d)} \
-        || ${@bb.utils.contains('MACHINEOVERRIDES', 'dual-gbe-cm4-mender', 'true', 'false', d)}; then
+
+    elif ${@'true' if 'seeed-reterminal' in d.getVar('MACHINEOVERRIDES').split(':') else 'false'} \
+        || ${@'true' if 'seeed-reterminal-mender' in d.getVar('MACHINEOVERRIDES').split(':') else 'false'} \
+        || ${@'true' if 'dual-gbe-cm4' in d.getVar('MACHINEOVERRIDES').split(':') else 'false'} \
+        || ${@'true' if 'dual-gbe-cm4-mender' in d.getVar('MACHINEOVERRIDES').split(':') else 'false'}; then
         grep -q "^dtoverlay=i2c3,pins_4_5$" $CONFIG || echo "dtoverlay=i2c3,pins_4_5" >> $CONFIG
         grep -q "^dtoverlay=reTerminal,tp_rotate=1$" $CONFIG || echo "dtoverlay=reTerminal,tp_rotate=1" >> $CONFIG
-    elif ${@bb.utils.contains('MACHINEOVERRIDES', 'seeed-recomputer-r100x-mender', 'true', 'false', d)} \
-        || ${@bb.utils.contains('MACHINEOVERRIDES', 'seeed-recomputer-r100x', 'true', 'false', d)}; then
+
+    elif ${@'true' if 'seeed-recomputer-r100x-mender' in d.getVar('MACHINEOVERRIDES').split(':') else 'false'} \
+        || ${@'true' if 'seeed-recomputer-r100x' in d.getVar('MACHINEOVERRIDES').split(':') else 'false'}; then
```

Comment:
I think I've already commented on how terrible this is. I'm glad I'm not the one who has to maintain it.